services/graphite/config.go

```diff
@@ -32,3 +56,19 @@ func NewConfig() Config {
 func (c *Config) LastEnabled() bool {
 	return c.NamePosition == strings.ToLower("last")
 }
+
+// ConsistencyAsEnum returns the enumerated write consistency level.
+func (c *Config) ConsistencyAsEnum() cluster.ConsistencyLevel {
+	switch strings.ToLower(c.ConsistencyLevel) {
+	case "any":
+		return cluster.ConsistencyLevelAny
+	case "one":
+		return cluster.ConsistencyLevelOne
+	case "quorum":
+		return cluster.ConsistencyLevelQuorum
+	case "all":
+		return cluster.ConsistencyLevelAll
+	default:
+		return cluster.ConsistencyLevelOne
+	}
+}
```

Comment:
A couple suggestions:

1. Can you move this to `cluster` so the other services can share it?

2. Can you rename it to `ParseConsistencyLevel(s string)`

3. Can you have it return an error if it can't be parsed? I don't think we should fail silently if someone types "`alll`". :)